### ISSUE TITLE
[$50] Gantt chart not displaying properly

### ISSUE THREAD

**Issue by jmgasper** [topcoder-platform/challenge-engine-ui#162]
On existing challenge pages (https://challenges.topcoder-dev.com/projects/3801/challenges/f46e37e2-a5ad-42bb-b82f-6d7ea51da0fb/edit)

The Gantt chart isn't displaying properly.  Each phase should be dependent on the phage before, but that isn't showing here.  This had been working previously, but the data structure in the response from the challenges API may have changed a bit so we need to investigate the changes.

**Comment by narekcat** [comment]
Hello @jmgasper . Can I work on this issue ?

**Comment by jmgasper** [comment]
@narekcat - Yes, that's fine.  Will set it to open shortly.  Please make sure to start on it right away.

**Comment by narekcat** [comment]
I see @Gauravseta self-assigned it, I'll try on next one, if they will be.

**Comment by jmgasper** [comment]
Contest https://www.topcoder.com/challenges/30123002 has been created for this ticket.<br/><br/>```This is an automated message for ghostar via Topcoder X```

**Comment by jmgasper** [comment]
Contest https://www.topcoder.com/challenges/30123002 has been updated - it has been assigned to gauravseta.<br/><br/>```This is an automated message for ghostar via Topcoder X```

**Comment by Gauravseta** [comment]
@narekcat sorry I just saw open for pickup label hence took it, missed your comment before that, hence giving it back to you.

@jmgasper

**Comment by jmgasper** [comment]
Contest https://www.topcoder.com/challenges/30123002 has been updated - it has been assigned to Narekvar90.<br/><br/>```This is an automated message for ghostar via Topcoder X```

**Comment by narekcat** [comment]
@Gauravseta Ok, thank you. @jmgasper I'll start to work on this issue.

**Comment by jmgasper** [comment]
@narekcat - Any update on this one?

**Comment by narekcat** [comment]
Hello @jmgasper.  There was a challenge: [Topcoder Challenge API - V5 Phases Business logic](https://www.topcoder.com/challenges/30113051). If I am right the main changes of challenge API was made according to that challenge specification. I can do appropriate changes in front-end, I just need additional hours to finish it.

**Comment by jmgasper** [comment]
@narekcat - Yeah, the main goal is just to make the Gantt chart show the relationships between phases again, like it used to.  If this takes a lot of effort, just let me know so I can bump the payment on this ticket.

**Comment by narekcat** [comment]
@jmgasper - I found out this changes:
1. challenge API returns `scheduledStartDate`, `scheduledEndDate`, `name` and other fields and we don't need to calculate start and end dates in front-end or get Phases for their names
2.  now `duration` saves in seconds not in hours, but in front-end calculation of start and end dates assume that duration is in hours
I almost finished with the front-end, I have changed challenge phases duration from hours to seconds for testing, by making request to development back-end.
